### DISCUSSION TITLE
Add recaptcha on Login Page using Keycloakify

### DISCUSSION THREAD

**Discussion by Darfito** [keycloakify/keycloakify#581] [Q&A]
Hello, I want to add recaptcha to my login page, but it seems the recaptcha didnt appear and help to setup up the site key and how to setup to my login.tsx

thx

**Comment by garronej** [comment]
Are you sure you want to set it on the login page?!  
This is very uncommon, you'd usually set it on the register page.

**Reply by denyFh** [reply]
Hi sorry for replying in this Q&A, but out of curiosity instead of recaptcha can we try to at least create like math formula field to the login page?
